templates/web/base/maps/noscript_map_wms.html

```diff
@@ -0,0 +1,32 @@
+<div class="noscript square-map__outer">
+    <div class="square-map__inner">
+        <div id="[% nsm_prefix %]drag">
+            [%- FOR row IN map.tiles -%]
+                [%- FOR tile IN row -%]
+                    [%- top_px = tile.row_offset * map.tile_size -%]
+                    [%- left_px = tile.col_offset * map.tile_size %]
+                    <[% map.img_type %]
```

Comment:
I don't know why the safe has disappeared from here compared to the WMTS, it's needed if it's an input type="image".